contributing.md

```diff
@@ -13,18 +13,33 @@ machine to develop Bazel and, when you've made a patch, how to submit it.</p>
 In general, we prefer contributions that fix bugs or add features (as opposed to
 stylistic, refactoring, or "cleanup" changes).
 
-Please check with us on the [dev
-list](https://groups.google.com/forum/#!forum/bazel-dev) before investing a lot
-of time in a patch. Meet Bazel maintainers and other contributors on
-[Slack](https://slack.bazel.build).
+### What issues can I work on?
+
+In general, if there is a Bazel team member assigned to an issue, someone is
+working on it. If not, you can use the priority level to as a rough guide.
```

Comment:
I'm not really convinced that the priority serves as a rough guide. I would use simpler guidelines, e.g.

* If it's assigned, someone might actively work on it. You may ask the assignee if you want to help or get an update.
* It it's not assigned, add a comment if you'd like to grab the issue.